db/re/item_db.conf

```diff
@@ -43668,7 +43668,7 @@ item_db: (
 },
 {
 	Id: 5012
-	AegisName: "Ph.D_Hat"
```

Comment:
We need to figure out a way to avoid editing constants. I'll see what I can do with libconfig, otherwise we can fall back to the list-of-lists approach in the drops list of the mob db